website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,33 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Select
+
+Multi-select enables users to select multiple rows from a dataset, significantly enhancing the efficiency of batch operations like delete, move, or update. This functionality is important for effective data management, especially in large datasets.
+
+!!! Info 
+
+Avoid using indeterminate states in individual rows. It's more effective at the global selection checkbox to reduce confusion.
+!!!
+
+<video width="100%" controls loop>
+  <source
+    src="/assets/components/table/multi-selection-interaction-01.mp4"
+    type="video/mp4"
+  />
+</video>
+
+### Suggested Application
+
+- For global selection and actions we recommend a master checkbox at the top of the table to select all dataset items in one click, simplifying bulk operations. The interface should allow operations like select all (rows) or deselct all (rows), and delete, or move to be applied to all selected rows for quick and efficient control across multiple pages.
```

Comment:
Couple of points here:

- This can be much more pointed and direct as well, shouldn't be worded as a recommendation because of how this feature works in the Ember component. The property that determines selection is `isSelectable`, if it's set to `{{true}}` at the top level of the table, the relevant UI elements will be rendered by default.
- This bullet point combines information about the intended UI, functionality, and how to handle bulk actions, which is pretty confusing. I think this should be decoupled a bit and each item addressed individually.